troubleshoot-lock-conflicts.md

```diff
@@ -230,4 +230,147 @@ When a pessimistic transaction has a deadlock, one of the transactions must be t
 
 Solutions:
 
-* The application needs to adjust transaction request logic when there too many deadlocks.
+* If it is difficult to confirm the cause of the deadlock, for v5.1 and later versions, you are recommended to try to query the `INFORMATION_SCHEMA.DEADLOCKS` or `INFORMATION_SCHEMA.CLUSTER_DEADLOCKS` system table to get the information of deadlock waiting chain. For details, see the [Deadlock errors](#deadlock-errors) section and the [`DEADLOCKS` table](/information-schema/information-schema-deadlocks.md) document.
+* If the deadlock occurs frequently, you need to adjust your application code to reduce such occurrences.
+
+### Use Lock View to troubleshoot issues related to pessimistic locks
+
+Since v5.1, TiDB supports the Lock View feature. This feature has several system tables built in `information_schema` that provide more information about the pessimistic lock conflicts and pessimistic lock waitings. For the detailed introduction of these tables, see the following documents:
+
+* [`TIDB_TRX` and `CLUSTER_TIDB_TRX`](/information-schema/information-schema-tidb-trx.md): Provides information of all running transactions on the current TiDB node or in the entire cluster, including whether the transaction is in the lock-waiting state, the lock-waiting time, and the digests of statements that have been executed in the transaction.
+* [`DATA_LOCK_WAITS`](/information-schema/information-schema-data-lock-waits.md): Provides the pessimistic lock-waiting information in TiKV, including the `start_ts` of the blocking and blocked transaction, the digest of the blocked SQL statement, and the key on which the waiting occurs.
+* [`DEADLOCKS` and `CLUSTER_DEADLOCKS`](/information-schema/information-schema-deadlocks.md): Provides the information of several deadlock events that have recently occurred on the current TiDB node or in the entire cluster, including the waiting relationship among transactions in the deadlock loops, the digest of the statement currently being executed in the transaction, and the key on which the waiting occurs.
+
+> **Warning:**
+>
+> Currently, this is an experimental feature. The definition and behavior of the table structure might have major changes in future releases.
+The following sections show the examples of troubleshooting some issues using these tables.
```

Comment:
```suggestion

The following sections show the examples of troubleshooting some issues using these tables.
```

Without the empty line, it's included in the quote section, but it's supposed not.